features/step_definitions/salt_steps.rb

```diff
@@ -1,9 +1,36 @@
 # Copyright 2015 SUSE LLC
+$testsuite_hostname = `hostname -f`.chomp
 
 When(/^I get a content of a file "(.*?)"$/) do |filename|
   $output = sshcmd("cat #{filename}")
 end
 
+When(/^I issue local command "(.*?)"$/) do |command|
+  $local_output = `#{command}`
+  $output = {:stdout => $local_output}
+end
+
```

Comment:
Those two steps are not specific to salt and therefore they belong in a different file. If you follow the previous suggestions, then the step for "And the salt-master should be listening on public port XX" may belong here, and this step could be implemented with the step above, but the step above is not Salt specific, and I bet there is a similar step somewhere else.